config/esp32/components/chip/component.mk

```diff
@@ -231,16 +125,8 @@ install-chip-with-gn : $(OUTPUT_DIR)
 	cd $(CHIP_ROOT) && PW_ENVSETUP_QUIET=1 . scripts/activate.sh && cd $(COMPONENT_PATH) && gn gen $(OUTPUT_DIR)
 	cd $(COMPONENT_PATH); ninja $(subst 1,-v,$(filter 1,$(V))) -C $(OUTPUT_DIR) esp32
 
-install-chip-with-automake: configure-chip
-	echo "INSTALL CHIP..."
-	MAKEFLAGS= make -C $(OUTPUT_DIR) --no-print-directory install
 
-ifneq ($(CHIP_BUILD_WITH_GN),n)
 install-chip: install-chip-with-gn
```

Comment:
Should we just rename `install-chip-with-gn` to `install-chip` and remove this line?

Reply:
Done.